apps/test/unit/p5lab/AnimationPicker/AnimationPickerBodyTest.js

```diff
@@ -9,7 +9,7 @@ import AnimationPickerListItem from '@cdo/apps/p5lab/AnimationPicker/AnimationPi
 import testAnimationLibrary from '../testAnimationLibrary.json';
 import {CostumeCategories} from '@cdo/apps/p5lab/spritelab/constants';
 import {PICKER_TYPE} from '@cdo/apps/p5lab/AnimationPicker/AnimationPicker';
-import AnimationUploadButton from '@cdo/apps/p5lab/AnimationPicker/AnimationUploadButton';
+import {UnconnectedAnimationUploadButton as AnimationUploadButton} from '@cdo/apps/p5lab/AnimationPicker/AnimationUploadButton';
```

Comment:
Do we still need this change?

Reply:
No. But I did add a few required props to the `defaultProps`.